modules/policy-security-regulation-compliance.adoc

```diff
@@ -14,7 +14,7 @@ Red Hat defines and follows a data classification standard to determine the sens
 
 [id="data-management_{context}"]
 == Data management
-{product-title} uses cloud provider services to help securely manage keys for encrypted data (AWS KMS and Google Cloud KMS). These keys are used for control plane data volumes which are encrypted by default. Persistent volumes for customer applications also use these cloud services for key management.
+{product-title} uses cloud provider services to help securely manage keys for encrypted data, AWS KMS and Google Cloud KMS. These keys are used for control plane data volumes which are encrypted by default. 
```

Comment:
You could change this line to something like:

`{product-title} uses cloud provider services such as AWS KMS and Google Cloud KMS to help securely manage keys for encrypted data. These keys are used for control plane data volumes which are encrypted by default. `

Reply:
You could also use "Key Management Service (KMS)" for the first use of "KMS" in each of the files.

Reply:
+1 and +1 to Paul's suggestions